src/helpers/vueI18next.js

```diff
@@ -0,0 +1,140 @@
+(function (global, factory) {
```

Comment:
Where did we copied this? Why can't we have this as package and need to copy/paste code?